index.html

```diff
@@ -0,0 +1,195 @@
+                    </svg>
+                    <h3 class="events">Exhibition</h3>
+                    <p class="event-text">
+                        Alongside the dynamic presenters and sessions on offer during the education summit.
+                    </p>
+                </div>
+            </div>
+            <h3 class="view-program">
+                <a href="#" class="a-text">View Full Program</a>
+            </h3>
+            <div class="register col-6 mx-auto">
+                <button class="reg-btn display-5 my-5 p-3">Register Yourselves</button>
+            </div>
+        </section>
+        <!-- Speaker section goes there -->
+        <section>
+            <h2 class="text-center mt-5">Featured Speakers</h2>
+            <div class="br-line"></div>
+                <article class="speaker-x mt-5" id="speakers">
+                </article>
+
+        </section>
+
+       <!-- Partner Section Goes There -->
+       <section class="partner-whole mt-5">
+        <h2 class="partners-title text-center pt-5">Partners</h2>
+        <div class="br-line"></div>
+        <div class="partners-main mx-5 text-center mt-5">
+           <div>
+            <img class="partners-pic" src="./images/Wellbeing+Web+(1).png" alt="">
+           </div> 
+           <div>
+            <img class="partners-pic" src="./images/Classroom+of+the+Future.png" alt="">
+           </div> 
+           <div>
+            <img class="partners-pic" src="./images/LIBRARIES+NEW+MILLENIUM.png" alt="">
+           </div> 
+           <div>
+            <img class="partners-pic" src="./images/Additional+Needs+Web+(1).png" alt="">
+           </div> 
+           <div>
+            <img class="partners-pic" src="./images/Literacy+Masterclass.png" alt="">
+           </div> 
+           <div>
+            <img class="partners-pic" src="./images/freeseminarsfloat.png" alt="">
+           </div>  
+        </div>
+    </section>
+        <!-- Footer goes there -->
+        <footer class="footer">
+            <h3 class="footer-1 text-center">
+                The National Educational <br>Summit
+            </h3>
+            <div>
+                <p class="footer-2 px-3">
+                    Originally Designed by Chindy Shin Behance,<br> some rights reserved 
+                </p>
+            </div>
+        </footer>
```

Comment:
- [ ] Please remove the "Partners" and the "Footer" section from the index page on mobile view as they don't exist in the original design.